ui/src/dashboards/components/DashboardsCardGrid.scss

```diff
@@ -16,37 +16,53 @@
 }
 
 .dashboards-card-grid {
-   display: grid;
-   grid-template-columns: 1fr;
-   grid-template-rows: 150px 150px 150px;
-   grid-column-gap: $ix-marg-a;
-   grid-row-gap: $ix-marg-a;
-   height: 100%;
-
-   .cf-resource-card {
-      margin-bottom: 0;
-   }
-
-   .cf-resource-description {
-      height: 60px;
+  padding-bottom: $cf-marg-d;
+  display: grid;
+  grid-template-columns: minmax(100%, 1fr);
+  grid-auto-rows: 152px;
+  grid-column-gap: $cf-marg-a;
+  grid-row-gap: $cf-marg-a;
+  height: 100%;
+
+  .cf-resource-card {
+    margin-bottom: 0;
+  }
+
+  .cf-resource-card--contents {
+    position: relative;
+  }
+
+  .cf-resource-editable-name {
+    position: absolute;
+    width: 100%;
+
+    .cf-resource-name--text {
       overflow: hidden;
-   }
+      text-overflow: ellipsis;
+    }
+  }
+
+  .cf-resource-description {
+    height: 60px;
+    margin-top: 22px;
+    overflow: hidden;
+  }
 }
 
 @media screen and (min-width: $cf-grid--breakpoint-sm) {
   .dashboards-card-grid {
-    grid-template-columns: 1fr 1fr;
+    grid-template-columns: minmax(50%, 1fr) minmax(50%, 1fr);
   }
 }
 
 @media screen and (min-width: $cf-grid--breakpoint-md) {
   .dashboards-card-grid {
-    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-columns: minmax(33.3333%, 1fr) minmax(33.3333%, 1fr) minmax(33.3333%, 1fr);
```

Comment:
`minmax` is like the CSS version of the interrobang

Also, `fr`???

Reply:
It's a CSS Grid thing. Stands for "fractional unit" and in this context controls how big the columns are. So `1fr 1fr 1fr` means divide up into 3 even columns.

Using `minmax()` ensures that the columns don't exceed that size